sys/include/net/gnrc/lwmac/types.h

```diff
@@ -90,12 +90,18 @@ extern "C" {
  */
 #define GNRC_LWMAC_RADIO_IS_ON               (0x04)
 
+/**
+ * @defgroup net_gnrc_lwmac_conf    GNRC LWMAC compile configurations
+ * @ingroup  net_gnrc_conf
+ * @{
+ */
 /**
  * @brief The default largest number of parallel timeouts in LWMAC
  */
 #ifndef GNRC_LWMAC_TIMEOUT_COUNT
 #define GNRC_LWMAC_TIMEOUT_COUNT             (3U)
 #endif
+/** @} */
```

Comment:
```suggestion
/**
 * @ingroup net_gnrc_lwmac_conf
 * @brief The default largest number of parallel timeouts in LWMAC
 */
#ifndef GNRC_LWMAC_TIMEOUT_COUNT
#define GNRC_LWMAC_TIMEOUT_COUNT             (3U)
#endif
```
That should be enough to add the macro to the group.